src/emqx.appup.src

```diff
@@ -874,11 +805,9 @@
      {load_module,emqx,brutal_purge,soft_purge,[]},
      {load_module,emqx_app,brutal_purge,soft_purge,[]},
      {load_module,emqx_message,brutal_purge,soft_purge,[]},
-     {load_module,emqx_limiter,brutal_purge,soft_purge,[]},
-     {load_module,emqx_hooks,brutal_purge,soft_purge,[]}]},
+     {load_module,emqx_limiter,brutal_purge,soft_purge,[]}]},
    {"4.3.0",
-    [{load_module,emqx_flapping,brutal_purge,soft_purge,[]},
-     {load_module,emqx_listeners,brutal_purge,soft_purge,[]},
```

Comment:
Just confirming: is the removal of these lines intentional?  Perhaps they were added erroneously before?

Reply:
yes, it's probably due to a bad compare base.